api/types/journal.go

```diff
@@ -1,4 +1,8 @@
 package apitypes
 
-// JournalIDHeader is the name of the HTTP Header used to provide a journal ID to assocate multiple actions.
+// JournalIDHeader is the name of the HTTP Header used to provide a activity ID to assocate multiple actions.
```

Comment:
Typo? I think the last part should be something like "provide a journal ID that Wash will use to record the activity".